lib/qs/job_handler.rb

```diff
@@ -57,6 +57,11 @@ def run_callback(callback)
 
     module ClassMethods
 
+      def timeout(value = nil)
+        @timeout = value if value
```

Comment:
@jcredding maybe do a `value.to_f` here?  Ensures you always have a float - that is the intention right?